ydb/core/base/blobstorage.h

```diff
@@ -841,6 +841,19 @@ struct TEvBlobStorage {
         EvControllerGroupDecommittedNotify          = 0x1003161e,
         EvControllerGroupDecommittedResponse        = 0x1003161f,
         EvControllerGroupMetricsExchange            = 0x10031620,
+        EvControllerProposeConfigRequest            = 0x10031821,
```

Comment:
You're adding new messages out-of-order -- this is group starting with 0x100316xx, not 18xx.